Example/AppClipExample/Shared/PaymentButton.swift

```diff
@@ -11,6 +11,8 @@ Abstract:
 A button that hosts PKPaymentButton from Apple's Fruta example app.
 */
 
+#if os(iOS)
```

Comment:
That's fair. I split it out because it was pretty messy when I did that (WKInterfaceObjectRepresentable is too different from UIViewRepresentable), but let me give the combined one another try.